### PR TITLE
add: `the-honkers-railway-launcher-bin`

### PR DESCRIPTION
**Pull request by vbrabandt2005**
## Progress 

- [x] Edit packagelist
- [x] Add initial pacscript
- [ ] Contact devs
- [x] Add maintainer to pacscript

### CODE REVIEW

**Comment by vbrabandt2005** [comment]
This pacscript is basically a modified version of an-anime-game-launcher-bin.pacscript but instead of installing that it installs **The Honkers railway launcher** instead.

I have added myself as the maintainer and is willing to update the script for every release (hopefully).

File: packages/the-honkers-railway-launcher-bin/the-honkers-railway-launcher-bin.pacscript [hunk 1]
```diff
@@ -0,0 +1,39 @@
+name="the-honkers-railway-launcher-bin"
+pkgname="the-honkers-railway-launcher"
```
Comment (Elsie19) [inline packages/the-honkers-railway-launcher-bin/the-honkers-railway-launcher-bin.pacscript:2]:
```suggestion
gives="the-honkers-railway-launcher"
```

File: packages/the-honkers-railway-launcher-bin/the-honkers-railway-launcher-bin.pacscript [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+pkgver="1.5.4"
+pkgdesc="The Honkers Railway launcher for Linux with automatic patching and telemetry disabling "
+repology=("project: the-honkers-railway-launcher")
+url="https://github.com/an-anime-team/the-honkers-railway-launcher/releases/download/${pkgver}/honkers-railway-launcher"
+hash="c7ce99c94a629089caa1a7d22ac68dbddfb7a7e81c1e9f447b1b800f7ecf84ee"
+depends=("xdelta3")
+maintainer="vbrabandt2005 <https://github.com/vbrabandt2005>"
+gives="${pkgname}"
```
Comment (Elsie19) [inline packages/the-honkers-railway-launcher-bin/the-honkers-railway-launcher-bin.pacscript:10]:
```suggestion
```

File: packages/the-honkers-railway-launcher-bin/the-honkers-railway-launcher-bin.pacscript [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+incompatible=("ubuntu:bionic" "ubuntu:focal" "ubuntu:jammy" "ubuntu:kinetic" "debian:stretch" "debian:buster" "debian:bullseye")
+breaks=("${pkgname}" "${pkgname}-deb" "${pkgname}-app" "${pkgname}-git")
```
Comment (Elsie19) [inline packages/the-honkers-railway-launcher-bin/the-honkers-railway-launcher-bin.pacscript:12]:
```suggestion
breaks=("${gives}" "${gives}-deb" "${gives}-app" "${gives}-git")
```

File: packages/the-honkers-railway-launcher-bin/the-honkers-railway-launcher-bin.pacscript [hunk 4]
```diff
@@ -0,0 +1,39 @@
+name="the-honkers-railway-launcher-bin"
+pkgname="the-honkers-railway-launcher"
+pkgver="1.5.4"
+pkgdesc="The Honkers Railway launcher for Linux with automatic patching and telemetry disabling "
```
Comment (Elsie19) [inline packages/the-honkers-railway-launcher-bin/the-honkers-railway-launcher-bin.pacscript:4]:
```suggestion
pkgdesc="The Honkers Railway launcher for Linux with automatic patching and telemetry disabling"
```

**Review by Elsie19 — CHANGES_REQUESTED** [review]

**Comment by vbrabandt2005** [comment]
@Elsie19, I had committed your suggestion into the script, I had to modify line19 in order to make it work, I can confirm the script does work locally on my PC, but I'm now receiving `build-local.sh: line 60: pkgname: parameter null or not set` in the failed checks, I'm actually kinda confused now.

**Comment by Elsie19** [comment]
> @Elsie19, I had committed your suggestion into the script, I had to modify line19 in order to make it work, I can confirm the script does work locally on my PC, but I'm now receiving `build-local.sh: line 60: pkgname: parameter null or not set` in the failed checks, I'm actually kinda confused now.

So we currently have breaking changes in the develop branch, where one of those changes is converting `$name->$pkgname`. If you are getting these errors locally, that means you are using `develop` with breaking changes on a repo targeted at the current pacstall version. If so, you should move back to `pacstall:master`.

If you are talking about the CI runs, it's the same problem as above, just ignore them though.

**Review by Elsie19 — APPROVED** [review]

**Pull request merged**